R/calc_match_person_db.R

```diff
@@ -58,7 +58,7 @@ calc_match_person_db <- function(df_one, id_one, forename_one, surname_one, dob_
   }
 
   # show warning prompt to user to make sure they have applied the formatting functions
-  cat("\nWARNING: Input datasets should have been formatted using available functions and output saved:\n\n")
+  cat("\nREMINDER: Have input datasets been formatted using available functions and output saved?:\n\n")
```

Comment:
Tweaked message as thought the warning could be interpreted by the user as thinking that it was telling them that they hadn't done the formatting. 

Wording as more of a reminder might prevent any confusion as this will always be shown, regardless of if they have just applied the formatting or not